src/ert/_c_wrappers/job_queue/queue.py

```diff
@@ -84,6 +85,25 @@
     "JOB_QUEUE_UNKNOWN": _FM_STEP_UNKNOWN,
 }
 
+_queue_state_to_pbuf_type_map = {
+    "JOB_QUEUE_NOT_ACTIVE": "STEP_WAITING",
```

Comment:
Should these queue_state (enum) strings be defined in some sensible place , like e.g. `job_status_type_enum.py`

Reply:
I think it's a good idea. Will do so...